### ISSUE TITLE
Update folding @ home

### ISSUE THREAD

**Issue by koppor** [dtgm/chocolatey-packages#488]
The [Folding at Home project](https://foldingathome.org/) supports research for the #coronavirus. The version available at https://download.foldingathome.org/releases/public/release/fah-installer/windows-10-32bit/v7.5/fah-installer_7.5.1_x86.exe should be offered via Chocolatey.

The current version at Chocolatey is 7.4.4 (see https://chocolatey.org/packages/fah).

**Comment by tunisiano187** [comment]
Done, but he isn't the maintainer anymore

**Comment by gep13** [comment]
@tunisiano187 can you send in another PR to remove the fah files from this repository?  Thanks

**Comment by tunisiano187** [comment]
Done

**Comment by gep13** [comment]
@tunisiano187 thank you!  We can close out this issue once your PR is merged.

**Comment by tunisiano187** [comment]
I've changed the Pull request

**Comment by gep13** [comment]
Perfect, thanks!
